packages/financial-templates-lib/src/logger/Transports.ts

```diff
@@ -45,7 +45,11 @@ export function createTransports(transportsConfig: TransportsConfig = {}): Trans
     const { LoggingWinston } = require("@google-cloud/logging-winston");
     transports.push(new LoggingWinston());
     if (!require("@google-cloud/trace-agent").get().enabled) require("@google-cloud/trace-agent").start();
-  } else if (transportsConfig.createConsoleTransport != undefined ? transportsConfig.createConsoleTransport : true) {
+  } else if (
+    transportsConfig.createConsoleTransport != undefined
+      ? transportsConfig.createConsoleTransport
+      : true || process.env.LOG_TO_CONSOLE == "true"
```

Comment:
Doesn't this always resolve to `true`?

```
true || process.env.LOG_TO_CONSOLE == "true"
```

Reply:
I'm a little confused here as well.

Reply:
I've refactored this. sorry it was overly verbose before!